octopose/local_deploy.py

```diff
@@ -35,15 +35,15 @@ def __init__(self, verbose):
 
     def invoke_deploy(self, step_path):
         """invoke_deploy start a deploy for a given powershell script (*Deploy.ps1)"""
+        print("- {0}".format(step_path))
```

Comment:
if your adding extra logging it might be worth allowing people to set verbose mode for extra context and not getting spammed normally. See context logger comment on other PR.

Reply:
Good point, though this isn't actually extra, it's just moved from line 39 (below)